### PR TITLE
Add execution bit accidentally dropped in #4106

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Title

image

### CODE REVIEW

**Review by yubiuser — DISMISSED** [review]

**Comment by PromoFaux** [comment]
Test run before setting execution bit on the two scripts with it missing in advanced/Scripts: https://github.com/pi-hole/pi-hole/runs/3797414113?check_suite_focus=true

**Comment by PromoFaux** [comment]
OK, I'm happy with this now (finally)

Tests failing look something like this.

image

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -5,21 +5,44 @@ on:
...
+      uses: actions/checkout@v2
+    -
+      name: Run Smoke Tests
+      run: |
+        # Ensure scripts in repository are executable
+        IFS=$'\n';
+        for f in $(find . -name '*.sh'); do if [[ ! -x $f ]]; then echo "$f is not executable" && FAIL=1; fi ;done
+        unset IFS;
+        # If FAIL is 1 then we fail.
+        [[ $FAIL == 1 ]] && exit 1 || echo "Smoke Tests Passed"
```
Comment (yubiuser) [inline .github/workflows/test.yml:23]: I'm a bit irritated by the `||` (OR).
But I guess it reads

```
If [[ $FAIL == 1 ]]; then
  exit 1
else
  echo "Smoke Tests Passed"
fi
```

Comment (PromoFaux) [reply .github/workflows/test.yml:23]: Yep, that's exactly what it says. I've not experimented with multi-line statements in a `run` section yet, but didn't really need to as this works just as well.

**Review by yubiuser — APPROVED** [review]

**Pull request merged**